python/ppml/example/fgboost_regression/fgboost-tutorial.md

```diff
@@ -27,8 +27,15 @@ First, import the package and initilize FL Context.
 from bigdl.ppml.fl import *
 from bigdl.ppml.fl.algorithms.fgboost_regression import FGBoostRegression
 import pandas as pd
+```
 
-init_fl_context()
+Party 1:
+```python
+init_fl_context('1')
+```
+Party 2:
+```python
+init_fl_context('2')
```

Comment:
I don't think you need to show the same code twice; maybe just
```
init_fl_context(client_id=id)
```

Reply:
I am considering some extensions of markdown generating HTML, which can create a tab-like style e.g. https://bookdown.org/yihui/rmarkdown-cookbook/html-tabs.html.

And I think readthedoc supports this feature?